### ISSUE TITLE
How can we choose epoch?

### ISSUE THREAD

**Issue by vhnamgau** [TuSimple/simpledet#311]
It is difficult to choose the suite epoch for stopping training. Could you please give us some document regarding this task.
Thanks

**Comment by xchani** [comment]
You may use TensorBoard to monitor the loss while training, and stop as it going steady. Here is the link of setting up: [TensorBoard](https://github.com/TuSimple/simpledet/blob/master/doc/TENSORBOARD.md).

**Comment by vhnamgau** [comment]
Thanks Chenxia Han for your answer.
TenserBoard is a tool to monitor : RcnnL1, RcnnAcc, RpnL1,RpnAcc. But how can I choose the epoch to stop training. We also need more document regarding RcnnL1, RcnnAcc, RpnL1,RpnAcc.

**Comment by huangzehao** [comment]
Hi, you can simply choose the last epoch. Since it is hard for network to overfit a large dataset like COCO.  Training longer and you will get better performance.
